### ISSUE TITLE
Incorrect response code logged into access.log for publish requests

### ISSUE THREAD

**Issue by turchanov** [slact/nchan#294]
Incorrect response code logged into access.log for publish requests. A client receives either 201 or 202 but in nginx access log the request is always logged with code 200.

To reproduce one needs some basic config 
```
http {
  listen 8080;
  server {
    location /pub {
      nchan_publisher;
      nchan_channel_group   test;
      nchan_channel_id   777;
    }
}
```
**curl -v http://127.0.0.1:8080/pub**
shows response code '202 Accepted' but nginx access log shows 
**"POST /pub HTTP/1.0" 200**

We need correct response code to be logged into access log.

**Comment by slact** [comment]
Fixed in  c586db3 . I also fixed incorrectly-logged subscriber response codes in  c586db3 .

**Comment by turchanov** [comment]
Applied your fix. Verified that correct codes logged into access.log. Thanks!
